lua/harpoon/list.lua

```diff
@@ -224,9 +224,16 @@ function HarpoonList:get(index)
     return self.items[index]
 end
 
-function HarpoonList:get_by_display(name)
-    local displayed = self:display()
-    local index = index_of(displayed, #displayed, name)
+function HarpoonList:get_by_path(path)
+    path = vim.loop.fs_realpath(vim.fs.normalize(path))
+    local index = index_of(self.items, self:length(), path, {
+        equals = function(a, b)
+            if b == nil then
+                return false
+            end
+            return a == b.value
```

Comment:
a == b.value

i don't like this

can we pass in a list_item and just use a.value == b.value?